dask/dataframe/io/tests/test_parquet.py

```diff
@@ -3636,3 +3636,15 @@ def test_ignore_metadata_file(tmpdir, engine, gather_statistics):
                 engine=engine,
                 ignore_metadata_file=True,
             )
+
+
+def test_extra_file(tmpdir, engine):
+    tmpdir = str(tmpdir)
+    df = pd.DataFrame({"a": range(100), "b": ["dog", "cat"] * 50})
+    ddf = dd.from_pandas(df, npartitions=2)
+    ddf.to_parquet(tmpdir, engine=engine)
+    open(os.path.join(tmpdir, "_SUCCESS"), "w").close()
+    open(os.path.join(tmpdir, "part.0.parquet.crc"), "w").close()
+    os.remove(os.path.join(tmpdir, "_metadata"))
+    out = dd.read_parquet(tmpdir)
```

Comment:
```suggestion
    out = dd.read_parquet(tmpdir, engine=engine)
```
This is always testing "fastparquet"